includes/settings/controls.php

```diff
@@ -127,7 +127,7 @@ private static function checkbox_list( array $field ) {
 		foreach ( $field['options'] as $option_key => $option_value ) :
 			?>
 			<label>
-				<input type="checkbox" name="<?php echo $field['id']; ?>[]" value="<?php echo $option_key; ?>"<?php checked( in_array( $option_key, $old_value ), true ); ?> />
+				<input type="checkbox" name="<?php echo esc_attr( $field['id'] ); ?>[]" value="<?php echo esc_attr( $option_key ); ?>"<?php checked( in_array( $option_key, $old_value ), true ); ?> />
 				<?php echo $option_value; ?>
```

Comment:
Please be aware that I'm pretty sure you want this to be:

```suggestion
				<?php echo esc_html( $option_value ); ?>
```

Alternatively it could run through `wp_kses_post()`